packages/debugger/style/breakpoints.css

```
@@ -32,6 +32,13 @@
   color: var(--jp-error-color1);
 }
 
+.jp-DebuggerBreakpoint-marker-selected {
+  font-size: 20px;
+  padding-right: 5px;
+  content: '◉';
```

Your PR follows the existing convention, but I am somewhat confused as to `◉` is specified in both `content` in CSS and in the React component.

In any case, my suggestion would be to work on accessibility here a little bit: this PR adds a a visual distinction between some markers so it should be reflected for screen readers.

```suggestion
  content: '◉' / 'Selected breakpoint';
```

Now we do not like CSS `content` not because it does not support accessible text (it does, 
https://www.stefanjudis.com/today-i-learned/css-content-accepts-alternative-text/) but because we cannot translate these accessible strings to target language.

If setting `content` is not required, ideally this would have a an accessibility hint via `aria-label` on the relevant div in react.

---

@krassowski Thanks for the review.

> Your PR follows the existing convention, but I am somewhat confused as to `◉` is specified in both `content` in CSS and in the React component.
I must admit that I am confused about it too.